app/navigators/AppNavigator.tsx

```diff
@@ -53,17 +63,53 @@ const AppStack = observer(function AppStack() {
 
   return (
     <Stack.Navigator
```

Comment:
the native back button doesn't work on navigation. When I press the ui back button it works but not the native back button.